site/en/docs/webstore/deceptive_installation_tactics/index.md

```diff
@@ -128,5 +129,12 @@ account, and possibly the termination of any associated publisher account(s).
 
 ## Is there a way to appeal this decision?
 
-Yes, appeals can be made through the instructions included in the notification email you will
-receive to the email address listed under your CWS developer account.
+Yes, appeals can be made by using the [CWS Support form][cws-support-form]. To learn more, see the [Complaint handling
+process FAQ][complaint-faq].
+
+[abuse-ad]: https://support.google.com/adspolicy/answer/6020954
+[ads-misrepresentation]: https://support.google.com/adspolicy/answer/6020955
+[complaint-faq]: /docs/webstore/complaint-faq
+[cws-support-form]: https://support.google.com/chrome_webstore/contact/one_stop_support
+[program-policies]: /docs/webstore/program_policies
```

Comment:
Does this list relate to the last question or to the whole page. It might need a heading such as `## See also` or an introduction that links it to the last question.